src/action-registry/action-registry.controller.ts

```diff
@@ -0,0 +1,31 @@
+import { Body, Controller, Get, Param, Patch, Post } from '@nestjs/common';
+import { ActionRegistryService } from './action-registry.service';
+import { CreateActionDto, UpdateActionDto } from './dto';
+
+@Controller('action-registry')
+export class ActionRegistryController {
+  constructor(private readonly actionRegistryService: ActionRegistryService) {}
+
+  @Get(':gameId')
+  async getManyByGameId(@Param('gameId') gameId: string) {
+    return this.actionRegistryService.getManyByGameId(gameId);
+  }
+
+  @Get(':actionId')
+  async getOneByActionId(@Param('actionId') actionId: string) {
+    return this.actionRegistryService.getOneByActionId(actionId);
+  }
+
+  @Post()
+  async createAction(@Body() dto: CreateActionDto) {
```

Comment:
There is no sense to add `Action` suffix as we already in `ActionRegistryController`
```suggestion
  async create(@Body() dto: CreateActionDto) {
```

Reply:
Solved in https://github.com/ggQuest/api-v2-nest/pull/155/commits/bd4a287b35da3835d352f89d08796ed77b441e06

Reply:
Solved in https://github.com/ggQuest/api-v2-nest/pull/155/commits/95f3a4946b4168316ed5336c418ae51631183201